WordPress/src/main/java/org/wordpress/android/ui/domains/management/newdomainsearch/composable/JetpackOutlinedButton.kt

```diff
@@ -0,0 +1,44 @@
+package org.wordpress.android.ui.domains.management.newdomainsearch.composable
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.ui.domains.management.success
+
+@Composable
+fun JetpackOutlinedButton(
+    text: String,
+    onClick: () -> Unit
+) {
+    OutlinedButton(
+        onClick = onClick,
+        shape = MaterialTheme.shapes.small.copy(CornerSize(36.dp)),
```

Comment:
I wonder whether we should make this button shape like the one used on the `MyDomainsScreen` (which is an m3 button shaped to match the composables used throughout the app). 🤔 

I.e.
```suggestion
        shape = RoundedCornerShape(4.dp),
```

I understand that this currently matches the designs in Figma, but maybe it makes sense to change it for consistency?

On a related note: I also wonder whether we should change the shape of the search input as well 🤔  (if so, we can tackle that in another PR - just wanted to share this thought here 😄 ).

Reply:
Totally makes sense!